homeassistant/components/google_drive/application_credentials.py

```diff
@@ -18,4 +19,5 @@ async def async_get_description_placeholders(hass: HomeAssistant) -> dict[str, s
         "oauth_consent_url": "https://console.cloud.google.com/apis/credentials/consent",
         "more_info_url": "https://www.home-assistant.io/integrations/google_drive/",
         "oauth_creds_url": "https://console.cloud.google.com/apis/credentials",
+        "redirect_url": config_entry_oauth2_flow.MY_AUTH_CALLBACK_PATH,
```

Comment:
But this is only the callback path no?

I think the full URL would be nice and to show the user the link in their situation

Reply:
Oh right, that one uses the current host the request is coming from. Are we sure that this is also not the case here?

Reply:
I don't think I follow. If the user has my enabled, this is the exact URL `https://my.home-assistant.io/redirect/oauth` they need to configure under Authorized redirect URI on the Google cloud project. This is for security so that not any random app could authorize. HA internally when wants to  authorize will use something like `https://accounts.google.com/o/oauth2/v2/auth?redirect_uri=https://my.home-assistant.io/redirect/oauth`

Reply:
No I get that, but if you have my disabled it looks wrong.

So I was wondering if these strings are also fetched via a request

Reply:
Ah you mean if they are fetched via an HTTP request to have access to the special header. Yes they are. I just noticed we actually have the hass as a param here so I was able to do https://github.com/home-assistant/core/pull/136906 instead.